RobertMrowiec/CarBIDEmber
# Integration tests

# Tasks:
- create myprofile integration tests
- create add-auction integration tests
- make few fixes in code like order etc
- change fetch to ember data where it's possible

# Testing scenario:
- ember test --server 
- select every integration tests
- check how it finishes successfully

# Preview
![image](https://user-images.githubusercontent.com/25043084/56351130-af132180-61cc-11e9-8e51-d6e6add0497c.png)
